### PR TITLE
RELEASE-516: Release PLF 5.0.0-RC02

### PR DESCRIPTION
**Pull request by mhbensghaier**

### CODE REVIEW

File: exo-platform/RELEASE-516.json [hunk 1]
```diff
@@ -0,0 +1,883 @@
...
+    "name":"platform-private-trial-distributions",
+    "git_organization":"exoplatform",
+    "labels":"plf",
+    "maven_property_version":"org.exoplatform.platform.distributions.private.trial.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 13
+    },
+    "release":{
+      "branch":"stable",
```
Comment (vsellier) [inline exo-platform/RELEASE-516.json]: the stable branches must be something like  ``stable/<name>``
where name = 5.0.x for project with a version=5.0.0-RC02
``1.3.x`` for version ``1.3.0-RC02``
and so on  

For project with no minor versions, the name is ``<version>.x``
for example for maven-depmgt-pom, the stable branch is ``stable/13.x``

Comment (mhbensghaier) [reply exo-platform/RELEASE-516.json]: branches has been modified as requested

File: exo-platform/RELEASE-516.json [hunk 2]
```diff
@@ -0,0 +1,883 @@
...
+    },
+    "git_organization":"exoplatform",
+    "labels":"addons,trial",
+    "maven_property_version":"addon.exo.web-conferencing.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/1.6.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-516.json]: s/1.6.x/1.1.x

File: exo-platform/RELEASE-516.json [hunk 3]
```diff
@@ -0,0 +1,883 @@
...
+    },
+    "git_organization":"exoplatform",
+    "labels":"addons",
+    "maven_property_version":"addon.exo.cmis.version",
+    "container": {
+      "image": "exoplatform/release:jdk8-maven33",
+      "step": 1
+    },
+    "release":{
+      "branch":"stable/1.6.x",
```
Comment (vsellier) [inline exo-platform/RELEASE-516.json]: s/1.6.x/5.0.x

File: exo-platform/RELEASE-516.json (hunk 2 above)
Comment (mhbensghaier) [reply exo-platform/RELEASE-516.json]: done

File: exo-platform/RELEASE-516.json (hunk 3 above)
Comment (mhbensghaier) [reply exo-platform/RELEASE-516.json]: done

**Pull request closed without merging**